src/main/java/apoc/refactor/GraphRefactoring.java

```diff
@@ -335,12 +336,22 @@ public void categorize(
         });
     }
 
-    private Node mergeNodes(Node source, Node target, boolean delete, RefactorConfig conf) {
+    private Node mergeNodes(Node source, Node target, boolean delete, Map<String, Object> config) {
         try {
             Map<String, Object> properties = source.getAllProperties();
             copyRelationships(source, copyLabels(source, target), delete);
+            RefactorConfig conf = new RefactorConfig(config);
             if (delete) source.delete();
             PropertiesManager.mergeProperties(properties, target, conf);
+
+            //check mergeRels parameter
+            Object value = config.get("mergeRels") == null ? false : config.get("mergeRels");
```

Comment:
this is really weird, that info should come cleanly out of RefactorConfig please move it there and add a dedicated method